python/paddle/v2/fluid/tests/test_layers.py

```diff
@@ -309,6 +309,34 @@ def test_multiplex(self):
             self.assertIsNotNone(out)
         print(str(program))
 
+    def test_detection_output(self):
```

Comment:
Write test in a new file: `tests/test_detection.py`.

Reply:
Done